website_google_tag_manager/__manifest__.py

```diff
@@ -16,10 +16,9 @@
     ],
     "data": [
         "views/res_config_settings_view.xml",
-        "views/website_views.xml",
         'views/website_templates.xml',
     ],
     'images': [
-        'static/description/icon.png',
```

Comment:
It's OK to remove the icon here, but don't remove the file per se, as it's the module icon (it's always automatically got from that path without being here declared).

Reply:
I don't remove it. Only was changed for new one :)

Reply:
This is saying the contrary:

https://github.com/OCA/website/pull/552/commits/4bad4da977df42c045b41012435226aa74012a32#diff-d6dd36ca45092f55ef21b2c115018a42

As said, there MUST BE a `static/description/icon.png` file in the module for having a module icon. If you add more images (which this section is for), great, but the other one is different.

Reply:
Of course and here is the icon file in his branch: https://github.com/Tecnativa/website/tree/12.0-mig-website_google_tag_manager/website_google_tag_manager/static/description

Reply:
It's very weird, as GitHub is saying that is removed:

![seleccion_004](https://user-images.githubusercontent.com/7165771/50280134-00dcb280-044c-11e9-9dd7-209ee658529f.png)

but indeed it's OK.

Reply:
Yeah, is about the image size, now is smaller. I hadn't noticed it before either.